### PR TITLE
upd(insync-deb): `3.5.4.50130` -> `3.7.6.50356`

### PR DESCRIPTION
**Pull request by Zahrun**

### CODE REVIEW

**Review by Henryws — CHANGES_REQUESTED** [review]
Fails on debian stable

**Comment by Zahrun** [comment]
> Fails on debian stable

ubuntu and debian debs have different hashes. Should I remove the hash check or support only ubuntu?

**Comment by Henryws** [comment]
> > Fails on debian stable
> 
> ubuntu and debian debs have different hashes. Should I remove the hash check or support only ubuntu?

Add cases for different hashes.

File: packages/insync-deb/insync-deb.pacscript [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
-url="https://d2t3ff60b2tol4.cloudfront.net/builds/${pkgname}_${version}-focal_amd64.deb"
+version="3.7.6.50356"
+repology=("project: insync")
+codename="focal"
+hash="428be8951c5dd4b35d68edbf029f94b05dd10e3452c49ecdc819d44fdeca0f6d"
+echo reading hash
+if [[ "$(lsb_release -is)" == "Debian" ]]; then
+  codename="buster"
+  hash="baf1b3c424171d092053c2dd667fa0f167db5a37cd0a94b5eac1bc78d412a1a0"
+  echo this is debian
```
Comment (Henryws) [inline packages/insync-deb/insync-deb.pacscript:11]: Remove this line

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/insync-deb/insync-deb.pacscript (hunk 1 above)
Comment (Zahrun) [reply packages/insync-deb/insync-deb.pacscript:11]: Yes sorry, this is a draft. I guess I should get a debian vm to try things out

**Comment by Zahrun** [comment]
From CI:
```
/var/lib/dpkg/info/insync.postinst: 135: pgrep: not found
Insync installation has finished. You may now start it.

/var/lib/dpkg/info/insync.postinst: 327: sysctl: not found
dpkg: error processing package insync (--configure):
 installed insync package post-installation script subprocess returned error exit status 127
```
It seems the CI debian install does not have procps package installed and that make it fail.
The installation works fine in my debian bullseye vm.

**Comment by Henryws** [comment]
Then add `procps` to `depends`

**Comment by Zahrun** [comment]
> Then add `procps` to `depends`

Pacstall silently ignores `depends` for -deb packages

**Comment by Henryws** [comment]
Oh, @D-Brox can you make a PR to fix that

**Review by Henryws — APPROVED** [review]

**Pull request merged**